app/models/user.rb

```diff
@@ -11,7 +12,8 @@ def self.from_omniauth(auth)
       name:   auth.info.name,
       login:  auth.extra.raw_info.login
     )
-    user.token = auth.credentials.token
+    user.token  = auth.credentials.token
+    user.role   = :user
```

Comment:
Maybe we should move a check for whitelisted organisations here?
Currently it feels dangerous: what if someone will use `from_omniauth` and then save a user object without a membership check? admin will be downgraded